lib/oli/delivery/metrics.ex

```diff
@@ -7,6 +7,9 @@ defmodule Oli.Delivery.Metrics do
   alias Oli.Delivery.Sections.SectionResource
   alias Oli.Delivery.Attempts.Core.{ResourceAccess, ActivityAttempt}
   alias Oli.Delivery.Attempts.Core
+  alias Oli.Delivery.Sections.{Enrollment, EnrollmentContextRole}
```

Comment:
Maybe we could add within the `Oli.Delivery.Sections.{Enrollment, EnrollmentContextRole}` aliases also the calls to the `SectionResource` and `ContainedPage` aliases found a few lines above, WDYT?